pkgs/by-name/op/openfortivpn-webview-qt/package.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
```

Comment:
Please reformat with the new formatter.

Reply:
I've reformat running `nix-shell` on the root of nixpkgs and then `nixfmt pkgs/by-name/op/openfortivpn-webview-qt/package.nix`.
the lines you highlighted did not change, but maybe you meant to just refer to the whole file

Reply:
Oh, you need to rebase on master first. I sort of wish the CI would say so... maybe I should get that fixed

Reply:
yes, I did had to rebase before `nixfmt` was available. I think it's ok on the latest commit 14dfe7b5335a, let me know otherwise